src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -3,8 +3,10 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 #---------------------------------------------------------------------------------------------
 
+import platform
 import random
 import string
+import urllib.request
```

Comment:
The CI build is failing because of this.
`urllib.request` is defined in Python 3 but not Python 2.

This should fix this:
`from six.moves.urllib.request import urlretrieve`
from: https://pythonhosted.org/six/#module-six.moves.urllib.request